docs/sources/flow/_index.md

```diff
@@ -78,18 +72,3 @@ prometheus.remote_write "default" {
 [configuration language]: {{< relref "./config-language/" >}}
 [reference documentation]: {{< relref "./reference" >}}
 
-## Current limitations
-
-The goal of Grafana Agent Flow is to eventually support the same use cases that
-Grafana Agent does today. Some functionality may be missing while Grafana Agent
-Flow is still in development:
-
-* An equivalent list of integrations
-* An equivalent to the scraping service
-
-## Provide feedback
-
```

Comment:
We could maybe keep this, no reason to throw it out right?

Reply:
I wanted to remove the feedback section because I'm worried it implies that Flow is still young or not production ready. 

I think people can rely on the "prove feedback" links elsewhere, like at the top level or at the project README. 

I'm going to merge this as-is to keep things moving, but if you're still unsure, let's discuss it and we can bring this back in a follow up PR :)